lib/model/model_test.go

```diff
@@ -868,6 +868,7 @@ func TestIssue5063(t *testing.T) {
 		conn.mut.Lock()
 		conn.closeFn = func(_ error) {}
 		conn.mut.Unlock()
+		defer m.Closed(c, errStopped) // to unblock m.Stop()
```

Comment:
Who blocks Stop() ?

Reply:
The line two above is a noop, therefore `model.Close` will deadlock ans it never receives the `Closed` call. However I now don't understand why I added this `closeFn` business in the first place.